staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -58,9 +58,13 @@ type SpdyRoundTripper struct {
 	// conn is the underlying network connection to the remote server.
 	conn net.Conn
 
-	// Dialer is the dialer used to connect.  Used if non-nil.
+	// Deprecated: Dialer is the dialer used to connect.  Used if non-nil and DialFuc is nil.
```

Comment:
you mean `DialContext` is nil , no?

Reply:
Yeah, I called it DialFunc originally.  I also considered DialWithContext, no strong preference on the name.